Content.Server/Movement/Systems/PullController.cs

```diff
@@ -128,8 +128,8 @@ private bool OnRequestMovePulledObject(ICommonSession? session, EntityCoordinate
             return false;
 
         // Cooldown buddy
-        if (_timing.CurTime < pullerComp.NextThrow)
-            return false;
+        //if (_timing.CurTime < pullerComp.NextThrow)
```

Comment:
remove it dont comment it

Reply:
Ah, wasn't sure what was good practice or not. I can remove it.